### ISSUE TITLE
"Re-create with solid"

### ISSUE THREAD

**Issue by tomByrer** [jherr/code-differ#1]
I did a little Remotion study a few months ago; I was trying to figure out how to use 1 Remotion file/repo to automatically create different aspect ratios for all social media outlets & screen sizes:
https://tombyrer.github.io/remotion-aspectratio-demo/

But I also found the React paradigm of props is a barrier to good DX.  Thought about SolidJS driven with [Motion One](https://motion.dev/), [Motion Canvas](https://motioncanvas.io/), &/or [TheatreJS](https://www.theatrejs.com/).  Also thought of using Svelte/Sveltosis which is more plain HTML.

Let me know if you want to team up!

**Comment by jherr** [comment]
Yeah, let's talk! You can connect with me on Discord jherr#2682 or on Twitter @jherr.

**Comment by tomByrer** [comment]
:)  Sent PM on Discord...

**Comment by jherr** [comment]
Hmmm... I didn't get anything. Let's just go with email jherr @ pobox . com.
